ipynb/examples/energy_meter/EnergyMeter_HWMON.ipynb

```diff
@@ -0,0 +1,357 @@
+      "02:13:41  INFO    :          RTApp - |  run_time  30000 [us], sleep_time  70000 [us]\n",
+      "02:13:41  INFO    :          RTApp - + phase_000008: duration 0.500000 [s] (5 loops)\n",
+      "02:13:41  INFO    :          RTApp - |  period   100000 [us], duty_cycle  25 %\n",
+      "02:13:41  INFO    :          RTApp - |  run_time  25000 [us], sleep_time  75000 [us]\n",
+      "02:13:41  INFO    :          RTApp - + phase_000009: duration 0.500000 [s] (5 loops)\n",
+      "02:13:41  INFO    :          RTApp - |  period   100000 [us], duty_cycle  20 %\n",
+      "02:13:41  INFO    :          RTApp - |  run_time  20000 [us], sleep_time  80000 [us]\n",
+      "02:13:45  INFO    :          WlGen - Workload execution START:\n",
+      "02:13:45  INFO    :          WlGen -    /root/devlib-target/bin/rt-app /root/devlib-target/ramp_00.json 2>&1\n",
+      "02:13:51  INFO    :          HWMon - Energy [             a53]: 9.036730\n",
+      "02:13:51  INFO    :          HWMon - Energy [             a57]: 2.050003\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create and RTApp RAMP task\n",
+    "rtapp = RTA(te.target, 'ramp', calibration=te.calibration())\n",
+    "rtapp.conf(kind='profile',\n",
+    "           params={\n",
+    "                'ramp' : Ramp(\n",
+    "                    start_pct =  60,\n",
+    "                    end_pct   =  20,\n",
+    "                    delta_pct =   5,\n",
+    "                    time_s    =   0.5).get()\n",
+    "          })\n",
+    "\n",
+    "# EnergyMeter Start\n",
+    "te.emeter.reset()\n",
+    "\n",
+    "rtapp.run(out_dir=te.res_dir)\n",
+    "\n",
+    "# EnergyMeter Stop and samples collection\n",
+    "nrg_report = te.emeter.report(te.res_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "02:13:51  INFO    : Collected data:\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[01;34m/data/lisa/results/EnergyMeter_HWMON\u001b[00m\r\n",
+      "├── energy.json\r\n",
+      "├── output.log\r\n",
+      "├── ramp_00.json\r\n",
+      "└── rt-app-ramp-0.log\r\n",
+      "\r\n",
+      "0 directories, 4 files\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "logging.info(\"Collected data:\")\n",
+    "!tree $te.res_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Power Measurements Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "02:13:52  INFO    : Measured channels energy:\n",
+      "02:13:52  INFO    : {'big': 9.036729999999807, 'little': 2.050003000000288}\n"
```

Comment:
Indeed, for the configured ramp... I would expect a bigger energy consumption on the LITTLE cluster...